plugins/modules/cloud/misc/terraform.py

```diff
@@ -335,6 +344,13 @@ def main():
         for f in variables_files:
             variables_args.extend(['-var-file', f])
 
+    if state == 'present' and check_destroy:
+        plan_file, needs_application, out, err, command = build_plan(command, project_path, variables_args, state_file,
+                                                                     module.params.get('targets'), state, plan_file)
+        if '- destroy' in out:
+            module.fail_json(msg="Aborting command because it would destroy some resources. Consider switching the 'check_destroy' to false to "
+                             "suppress this error")
```

Comment:
```suggestion
            module.fail_json(msg="Aborting command because it would destroy some resources. "
                                                "Consider switching the 'check_destroy' to false to suppress this error")
```